README.md

```diff
@@ -134,29 +134,75 @@ Sketches:
 --------------------------------------------------------------------------------
 <BASE64_ENCODED_DATA_POINTS|REFERENCE_TO_OTHER_DATA_FILE>
 ```
-
-Each Protocol has unique set of parameters. For example:
-- CPI
+Each Sync Protocol has an integer to identify it. The list of integers is as follows:
+1. CPISync
+2. CPISync_OneLessRound
+3. CPISync_HalfRound
+4. ProbCPISync
+5. InteractiveCPISync
+6. OneWayCPISync
+7. FullSync
+8. IBLTSync
+9. OneWayIBLTSync
+10. IBLTSetOfSets
+11. IBLTSync_Multiset
+12. CuckooSync
+13. BloomFilterSync
+14. MET_IBLTSync
+    
+Each Protocol has a unique set of parameters.
+ 
+CPI
 ```
 m_bar: <INT>
 bits: <INT>
 epsilon: <INT>
 partitions/pFactor(for InterCPISync): <INT>
 hashes: <true|false>
 ```
-- IBLT
+- m_bar: This is the maximum degree of the polynomial being used in the interpolation. It is analogous to the differences between the sets.
+- bits: Refers to the bit length used for encoding the coefficients of the polynomial.
```

Comment:
not coefficients - evaluations ... of the characteristic polynomial.  From the user's perspective, the only thing that is relevant is that this is the number of bits used to represent an element.